frontend/src/app/components/ExperimentCardList.js

```diff
@@ -39,7 +30,7 @@ export default class ExperimentCardList extends React.Component {
 
   render() {
     if (this.props.experiments.length === 0) {
-      return this.renderLoading();
+      return null;
```

Comment:
using this return null pattern throughout this commit in containers that used to return a loading state